main.nf

```diff
@@ -623,7 +625,7 @@ process dupradar {
     script: // This script is bundled with the pipeline, in NGI-RNAseq/bin/
     def paired = single ? 'FALSE' :  'TRUE'
     """
-    dupRadar.r $bam_md $gtf $paired
+    dupRadar.r $bam_md $gtf $paired ${params.rlocation}
```

Comment:
I think that this may print `false` if it's false, when we actually want an empty string (though I haven't tried this yet).

```groovy
def rlocation = params.rlocation ?: ''
"""
dupRadar.r $bam_md $gtf $paired $rlocation
"""
```